java/connector-node/connector-api/src/main/java/com/risingwave/connector/api/sink/SinkRow.java

```diff
@@ -16,10 +16,10 @@
 
 import com.risingwave.proto.Data;
 
-public interface SinkRow {
-    public Object get(int index);
+public interface SinkRow extends AutoCloseable {
```

Comment:
Curious why a row need to be closeable?

Reply:
according to offline discussion with @wenym1 , it can prevent memory leak.

Reply:
In future PR, we will process stream chunk via jni. Therefore we will hold a row handle in the returned SinkRow, and need explicit close to release the memory.